src/assets/images/index.ts

```diff
@@ -1,32 +1,34 @@
-const BlueGradientBackground = require("./blueGradientBackground.png")
-const BoxCheckedIcon = require("./boxCheckedIcon.png")
-const BoxUncheckedIcon = require("./boxUncheckedIcon.png")
-const CheckInCircle = require("./checkInCircle.png")
-const CodesAndPhone = require("./codesAndPhone.png")
-const ConcentricCircles = require("./concentricCircles.png")
-const Doctors = require("./doctors.png")
-const EmptyPathBackground = require("./emptyPathBackground.png")
-const ExclamationInCircle = require("./exclamationInCircle.png")
-const Export = require("./export.png")
-const ForeArrow = require("./foreArrow.png")
-const HomeActive = require("./homeActive.png")
-const HomeInactive = require("./homeInactive.png")
-const HugEmoji = require("./hugEmoji.png")
-const IsolatePathBackground = require("./isolatePathBackground.png")
-const MinnesotaMap = require("./minnesotaMap.png")
-const MultipleCrossPathBackground = require("./multipleCrossPathBackground.png")
-const PeopleHighFiving = require("./peopleHighFiving.png")
-const PeopleOnPhones = require("./peopleOnPhones.png")
-const PersonAndHealthExpert = require("./personAndHealthExpert.png")
-const PersonGettingNotification = require("./personGettingNotification.png")
-const PersonOnComputer = require("./personOnComputer.png")
-const PersonOnPhoneWithCode = require("./personOnPhoneWithCode.png")
-const PersonWithCodesInBackground = require("./personWithCodesInBackground.png")
-const PersonWithLockedPhone = require("./personWithLockedPhone.png")
-const PersonWithPhoneAndCode = require("./personWithPhoneAndCode.png")
-const PhonesSharingCodes = require("./phonesSharingCodes.png")
-const SingleCrossPathBackground = require("./singleCrossPathBackground.png")
-const WelcomeImage = require("./welcomeImage.png")
+import BlueGradientBackground from "./blueGradientBackground.png"
```

Comment:
are we able to use the es6 import syntax here? irc, this cause some typescript errors?

Reply:
Seems to work just fine!